.github/workflows/test.yml

```diff
@@ -323,8 +323,8 @@ jobs:
           ref: ${{ github.event.pull_request.head.sha || github.sha }}
           submodules: recursive
 
-      - name: foundry-install
-        uses: foundry-rs/foundry-toolchain@v1
+      - name: Setup Foundry
+        uses: ./.github/actions/setup-foundry
```

Comment:
duped by the install-cli which uses yarn-build-with-cache after it

Reply:
Fixed